src/frontend/src/service/api-service.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios';
+
+type MethodOptions =
+  | 'POST'
+  | 'post'
+  | 'GET'
+  | 'get'
+  | 'DELETE'
+  | 'delete'
+  | 'head'
+  | 'HEAD'
+  | 'put'
+  | 'PUT'
+  | 'patch'
+  | 'PATCH';
```

Comment:
I think we should standardize on either capitalized or uncapitalized method names and not allow both. I personally prefer lowercase names but it's not a strong preference; I'm open to either.

Reply:
I have not updated the MethodOptions to look as seen below.  However when I try to change my api-service.test.js tile to  a typescript file, the type check fails and it says "type string is not assignable to method options".  Would someone know how to fix this TypescriptError?
enum MethodOptions {
  post = "post",
  get = "get",
  delete = "delete",
  head = "head",
  put = "put",
  patch = "patch"
}

Reply:
I have now fixed the type error.

Reply:
@SteelFace87 sounds good, let me know when you're done with all new updates to the PR and I'll re-review the whole thing.